### ISSUE TITLE
Translated output are blurry

### ISSUE THREAD

**Issue by supritbansod** [LynnHo/CycleGAN-Tensorflow-Simple#36]
Hi Lynn Ho, 

I applied the tensorflow-2 cycleGAN implementation on my own dataset. The output translated images are blurry. I went through the paper of cycleGAN but could not find a parameter to be fine-tuned in case of blurriness. Do you have any idea how this can be solved? Also, how gradient penalty mode and gradient penalty weight parameters in the train.py script works?

**Comment by LynnHo** [comment]
@supritbansod Try to decrease cycle_loss_weight（or increase the adversarial loss weight, hard coded as 1） or train it with a sufficiently long time. But I am not sure whether it will work, maybe CycleGAN is not suitable for your data. 

I do not very understand your second question, you can find the code of gradient penalty here.
https://github.com/LynnHo/CycleGAN-Tensorflow-2/blob/1aa6398d918875a1cf25320881d09b7d9f3f63b8/tf2gan/loss.py#L86
